src/components/basic/FlatList/FlatList.tsx

```diff
@@ -10,7 +10,10 @@ import { useHasResponsiveProps } from '../../../hooks/useHasResponsiveProps';
 
 const StyledFlatList: any = makeStyledComponent(RNFlatList);
 
-export const FlatList = forwardRef((props: IFlatListProps, ref: any) => {
+const FlatListComponent = <ItemT extends any>(
```

Comment:
is the `extends any` necessary? 
```suggestion
const FlatListComponent = <ItemT>(
```